.travis.yml

```diff
@@ -9,7 +9,7 @@ env:
   global:
     - VERSION=1.1
     - NETWORK=mainnet
-    - CARDANO_SL_BRANCH=release/1.1.0
+    - CARDANO_SL_BRANCH=1.1.0-staging
```

Comment:
Why do we need a separate branch for staging on cardano side?

Reply:
@domenkozar, because there is no sure-fire way to determine if the current combination of versions will work -- so I was hesitant to make a PR to `cardano-sl`.  If the branch produces an installer that actually works -- then I think we're safe to make that PR.

At least that's my idea.

Reply:
Ah, sounds good! I would prefer another daedalus PR also for such experiments, since this is considered a staging de-facto usable installer.

Reply:
But we don't have usable staging installers yet -- _and_ this branch isn't supposed to be merged (unlike the Cardano one).  What purpose the extra PR would serve?

Reply:
Once I turn on mirroring they will be functional, regardless of this change :)

Reply:
@gromakovsky will correct this